netbox/dcim/views.py

```diff
@@ -458,7 +458,7 @@ class RackBulkDeleteView(PermissionRequiredMixin, BulkDeleteView):
 #
 
 class RackReservationListView(ObjectListView):
-    queryset = RackReservation.objects.all()
+    queryset = RackReservation.objects.all().select_related('rack__site')
```

Comment:
You can remove `.all()` when it's followed by another method; it's effectively a no-op.

Reply:
Fixed in 1ff7e11